jellyfin.ruleset

```diff
@@ -53,7 +63,11 @@
     <Rule Id="CA1822" Action="Info" />
     <!-- disable warning CA2000: Dispose objects before losing scope -->
     <Rule Id="CA2000" Action="Info" />
+    <!-- disable warning CA5394: Do not use insecure randomness -->
```

Comment:
This one doesn't look like something we'd want to blanket disable? Should probably be done on a case-by-case basis.

I'll defer this decision to someone with a bit more knowledge about the linting rules.

Reply:
TBH I agree, however I used the [jellyfin.ruleset from the server repo](https://github.com/jellyfin/jellyfin/blob/51a075a89533543ef3a35f7b85f90ec5fd4e743d/jellyfin.ruleset#L66) to update this one. That does not mean that it is flawless or that we should blanket use it everywhere.
However, I know to little about the linting rules myself to make the judgment so it ended up here :thinking:

Reply:
alright, if it came directly from server it's *probably* fine